### PR TITLE
Upgrade PTF image to python3

### PR DESCRIPTION
**Pull request by Yi-Tseng**
Test with https://github.com/stratum/fabric-tna/pull/135

### CODE REVIEW

File: tools/ptf/Dockerfile [hunk 1]
```diff
@@ -75,19 +74,19 @@ ARG GRPC_VER
...
+	python3-setuptools \
     git
 
 ENV PIP_DEPS \
-    git+https://github.com/p4lang/scapy-vxlan \
     git+https://github.com/p4lang/ptf \
     protobuf==$PROTOBUF_VER \
-    grpcio==$GRPC_VER
+    grpcio==$GRPC_VER \
+    scapy
```
Comment (ccascone) [inline tools/ptf/Dockerfile:86]: pin version for reproducible builds

File: tools/ptf/Dockerfile [hunk 2]
```diff
@@ -7,8 +7,9 @@
 
 ARG GRPC_VER=1.26
 ARG PROTOBUF_VER=3.12
+ARG SCAPY_VER=2.4.3
 
-FROM python:2.7.13 as proto-deps
+FROM python:3.5 as proto-deps
```
Comment (ccascone) [inline tools/ptf/Dockerfile:12]: Python 3.5 has reached end-of-life
```suggestion
FROM python:3.9 as proto-deps
```

File: tools/ptf/Dockerfile [hunk 3]
```diff
@@ -96,15 +97,15 @@ FROM bitnami/minideb:stretch
...
+    python3 \
+    python3-setuptools \
     iproute2 \
     tcpdump \
     netbase
 
 RUN install_packages $RUNTIME_DEPS
 
-COPY --from=proto-deps /output /usr/lib/python2.7/dist-packages
+COPY --from=proto-deps /output /usr/lib/python3.5/dist-packages
```
Comment (ccascone) [inline tools/ptf/Dockerfile:108]:
```suggestion
COPY --from=proto-deps /output /usr/lib/python3.9/dist-packages
```

**Review by ccascone — APPROVED** [review]
LGTM. Can this be merged?

**Comment by Yi-Tseng** [comment]
> LGTM. Can this be merged?

let me change it to 3.9 and we should be able to merge it

**Pull request merged**